ssl/t1_lib.c

```diff
@@ -1466,6 +1703,13 @@ static int sigalg_security_bits(SSL_CTX *ctx, const SIGALG_LOOKUP *lu)
         else if (lu->sigalg == TLSEXT_SIGALG_ed448)
```

Comment:
The start of this function calls `tls1_lookup_md`....does that actually work for provider based sigalgs that include a digest if the digest is not a normal built in one?

Reply:
Yes, if the digest comes with the sigalg provider (see documentation update). 
No, if the digest is loaded by yet another provider. But as far as I can tell this didn't work before this PR either ("We ignore this for now"): See https://github.com/openssl/openssl/blob/4ff66347f0ab3c054c5622dd862f36c731c889ed/ssl/t1_lib.c#L1175-L1187

Reply:
@mattcaswell, please let me know if there's something I can do to get this conversation resolved.

Reply:
I'd go with "We ignore this for now"

It's well known among at least a few of us in the team that libssl code needs a pretty deep review and refactoring re provider supported algorithms.  This is one of those bits.
I would leave this for another PR for now.

Reply:
Ok...we'll go with "ignore this for now".